incubator/elasticsearch/templates/rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
we need to add the boolean field `rbac.create` to enable/disable this resource creation. see https://github.com/kubernetes/charts/blob/master/stable/cert-manager/templates/rbac.yaml#L1 and https://github.com/kubernetes/charts/blob/master/stable/cert-manager/values.yaml#L13-L15

Reply:
Why? I mean it's mandatory for the plugin. ES release won't work without RBAC

Reply:
are you sure RBAC is required for the plugin to work?

My suggestion was based on the observation that `helm install` failed while i tried to deploy the chart on a GKE cluster. I guess it was because of the typo's that were present in `rolebinding.yaml` file earlier.

Reply:
Yes @sameersbn ! RBAC is necessary since the plugin needs permissions to consult the K8s API in order to discover other ES nodes

Reply:
We might need to update the "Prerequisites" section in the **README.md** as we did for "PV provisioner"